Libs/MRML/Core/vtkMRMLVolumeSequenceStorageNode.cxx

```diff
@@ -425,6 +425,7 @@ int vtkMRMLVolumeSequenceStorageNode::WriteDataInternal(vtkMRMLNode *refNode)
   // Use here the NRRD Writer
 #if Slicer_VERSION_MAJOR > 4 || (Slicer_VERSION_MAJOR == 4 && Slicer_VERSION_MINOR >= 9)
   vtkNew<vtkTeemNRRDWriter> writer;
+  writer->SetForceRangeAxis(true);
```

Comment:
Describe in a comment why this is required. It would be even better to add a simple test for it.